packages/excalidraw/components/TTDDialog/MermaidToExcalidraw.tsx

```diff
@@ -54,7 +54,9 @@ const MermaidToExcalidraw = ({
       mermaidToExcalidrawLib,
       setError,
       mermaidDefinition: deferredText,
-    }).catch(() => {});
+    }).catch((err) => {
+      console.error("Failed to parse mermaid definition", err);
```

Comment:
this will log error (and to Sentry) on each keystroke of invalid diagram which happens a lot if you edit/write your mermaid diagrams

Reply:
That's a good point, I will do this only for development mode then